content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Master Controller Cheat-Sheet"
+linkTitle: "Cheat-Sheet MasterController"
```

Comment:
spelling: cheat sheet (no hyphen), mastercontroller (one word, lowercase, that's how it should be throughout the doc)